scheduler/CHANGELOG.md

```diff
@@ -4,6 +4,20 @@ All notable changes to this project will be documented in this file
  
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.7.1] - 2017-11-07
+### Added
+- Added support for killing a group of jobs, from @DaoWen
+- Added sysouts to get job output closer to Mesos' CommandExecutor, from @shamsimam
+- Added metrics support for usage of /list, from @dposada
+- Added support for groups on retry endpoint, from @DaoWen
+- Added support for configurable progress file in Cook Executor, from @shamsimam
+- Added support for configurable environment passed to Cook Executor, from @shamsimam
```

Comment:
Should these two be in a separate changelog for the executor?

Reply:
Just put the last one as it reflects changes in the scheduler.

I'd rather avoid a changelog for the executor for now.